curp-external-api/src/cmd.rs

```diff
@@ -2,14 +2,23 @@ use std::{fmt::Display, hash::Hash};
 
 use async_trait::async_trait;
 use engine::Snapshot;
+use prost::DecodeError;
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 
 use crate::LogIndex;
 
 /// Command to execute on the server side
 #[async_trait]
 pub trait Command:
-    Sized + Sync + Send + DeserializeOwned + Serialize + std::fmt::Debug + Clone + ConflictCheck
+    Sized
```

Comment:
We can remove the `Sized` trait bound here in that `PbSerialize` has already met the `Sized` bound.

Reply:
done